app-web/forgejo/autobuild/build

```diff
@@ -0,0 +1,7 @@
+abinfo "Building Forgejo"
+unset LDFLAGS
```

Comment:
Why?

Reply:
does not compile with our default LDFLAGS, added a comment to explain this.